web-common/src/features/dashboards/url-state/convert-partial-explore-state-to-url-params.ts

```diff
@@ -53,30 +54,23 @@ export function getCleanedUrlParamsForGoto(
     timeControlsState,
   );
 
-  // Remove params with default values
-  [...stateParams.entries()].forEach(([key, value]) => {
-    const defaultValue = defaultExploreUrlParams.get(key);
-    if (
-      (defaultValue === null && value !== "") ||
-      (defaultValue !== null && value !== defaultValue)
-    ) {
-      return;
-    }
-    stateParams.delete(key);
-  });
+  const strippedUrlParams = stripDefaultUrlParams(
```

Comment:
Naming: I'd expect the "strippedURLParams" to be the URL params _removed_, not the ones that are retained